assets/source/setup-guide/app/helpers/get-context-by-path.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { getPath } from '@woocommerce/navigation';
+
+/**
+ * Returns the formatted context based on the path.
+ *
+ * @return {string} The path in snake case format
+ * @throws Exception if getPath is not working, returns an empty string as a result
```

Comment:
❓ 
This function won't throw an exception.

Reply:
It's outdated. See comments in https://github.com/woocommerce/pinterest-for-woocommerce/pull/295#pullrequestreview-833736094 

However you're right, I guess at the beginning Iw as adding a throw and then I change my opinion and forgot to update the docs.

Reply:
See also https://github.com/woocommerce/pinterest-for-woocommerce/pull/295/commits/e51fbfd87d6b2a2e016196ee9ae73a2248a399b3 where I fixed all related to this function